catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/platform/transformer/MiroTransformerFeatureTest.scala

```diff
@@ -72,4 +72,96 @@ class MiroTransformerFeatureTest
+                hybridRecordNotificationMessage(
+                  message = createValidMiroTransformableJson(
+                    MiroID = "L0011975",
+                    MiroCollection = "images-L",
+                    data = """
+                      {
+                          "image_cleared": "Y",
+                          "image_copyright_cleared": "Y",
+                          "image_credit_line": "Wellcome Library, London",
+                          "image_image_desc": "Antonio Dionisi",
+                          "image_innopac_id": "12917175",
+                          "image_library_dept": "General Collections",
+                          "image_no_calc": "L0011975",
+                          "image_phys_format": "Book",
+                          "image_tech_file_size": [
+                              "5247788"
+                          ],
+                          "image_title": "Antonio Dionisi",
+                          "image_use_restrictions": "CC-BY"
+                      }
+                    """
+                  ),
+                  sourceName = "miro",
+                  s3Client = s3Client,
+                  bucket = storageBucket
+                )
+              val miroHybridRecordMessage2 =
+                hybridRecordNotificationMessage(
+                  message = createValidMiroTransformableJson(
+                    MiroID = "L0023034",
+                    MiroCollection = "images-L",
+                    data =
+                      """
+                      {
+                          "image_cleared": "Y",
+                          "image_copyright_cleared": "Y",
+                          "image_image_desc": "Use of the guillotine",
+                          "image_innopac_id": "12074536",
+                          "image_keywords": [
+                              "Surgery"
+                          ],
+                          "image_library_dept": "General Collections",
+                          "image_no_calc": "L0023034",
+                          "image_tech_file_size": [
+                              "5710662"
+                          ],
+                          "image_title": "Greenfield Sluder, Tonsillectomy..., use of guillotine",
+                          "image_use_restrictions": "CC-BY"
+                      }
+                    """
+                  ),
+                  sourceName = "miro",
+                  s3Client = s3Client,
+                  bucket = storageBucket
+                )
+              sqsClient.sendMessage(
+                queue.url,
+                JsonUtil.toJson(miroHybridRecordMessage1).get
+              )
+
+              sqsClient.sendMessage(
+                queue.url,
+                JsonUtil.toJson(miroHybridRecordMessage2).get
+              )
+              eventually {
+                val works = getMessages[UnidentifiedWork](topic)
+                works.distinct.length shouldBe >=(2)
+
```

Comment:
I don't think this is wrong, but it's not obvious to me why there can be more than, rather than exactly, 2 works after processing.

Reply:
Hmm. Possibly for more-than-once delivery, but we'd get duplicate SNS messages – not duplicate works.

I think we should change this to `shouldBe 2`.

Reply:
yeah you're right. It's a copy pasta error from test where we have to account for at least once delivery